app/charts/shared/containers.tsx

```diff
@@ -22,6 +22,8 @@ export const useStyles = makeStyles<{}, {}, "chartContainer">(() => ({
   },
 }));
 
+const INTERACTIVE_TOGGLE_SPACING = 20;
```

Comment:
Nit: @noahonyejese it looks like the height is 20 on desktop, but 18 on mobile. This is a nit, but one way we could solve it would be to use:

```typescript
<Box
  component="foreignobject"
  width={...}
  sx={{ display: "flex", y: [18, 20], height: 24, textAlign: "right" }}
>
```

Also, maybe adding a little bit of padding to the bottom, like `height: 26` would make it less crowded? 🤔 

<img width="274" alt="Screenshot 2024-11-12 at 10 09 10" src="https://github.com/user-attachments/assets/46353931-aeaa-4e6c-8929-4bd4cb8f1f68">